wings/market/market_base.pyx

```diff
@@ -42,6 +42,10 @@ cdef class MarketBase(NetworkIterator):
             self.c_add_listener(event_tag.value, self.event_reporter)
             self.c_add_listener(event_tag.value, self.event_logger)
 
+    @property
+    def status_dict(self) -> Dict[str, bool]:
+        return {}
```

Comment:
should this throw error? if you do {}.values it raises an error "TypeError: 'builtin_function_or_method' object is not iterable"

Reply:
Calling {}.values() should return dict_values([])